server/text_generation_server/layers/attention/flashinfer.py

```diff
@@ -84,16 +84,16 @@ def use_prefill_with_paged_kv_state(
     token = prefill_with_paged_kv_state.set(state)
     try:
         state.plan(
-            qo_indptr=cu_seqlens,
-            paged_kv_indptr=indptr,
-            paged_kv_indices=block_tables,
-            paged_kv_last_page_len=last_page_len,
-            num_qo_heads=num_heads,
-            num_kv_heads=num_kv_heads,
-            head_dim=head_size,
-            kv_data_type=kv_dtype,
+            cu_seqlens,
+            indptr,
+            block_tables,
+            last_page_len,
+            num_heads,
+            num_kv_heads,
+            head_size,
+            page_size,
```

Comment:
I don't think moving away from kwargs is the right solution. The error show exactly why using kwargs is good - it catches changes to arguments. We should update to the latest flashinfer, see: https://github.com/huggingface/text-generation-inference/pull/3164/files

But a lot of the test outputs change after upgrading, I haven't had the time yet to go through them and see if they are all minor fluctuations or not.